### PR TITLE
Solaris Map Bugfixes

### PR DESCRIPTION
**Pull request by LambdaSucks**
# About the pull request

This fixes multiple instances of misdirected cameras throughout Solaris ridge, as well as multiple sections of the map with two or more APCs.

Fixes have also been made to multiple Solaris Ridge inserts, just fixing floating cameras.

Solaris Ridge map fixes:

Fixed the few floating cameras located throughout the map.

Fixed the double APC in Atmos.

Fixed the double APC in Southeast Lambda

Solaris Ridge insert fixes:

Prison Breakout - Fixed floating cameras.

Cargo Containers - Fixed floating cameras.

Admin Party - Fixed floating cameras.

# Explain why it's good for the game

This restores the visual continuity of the map, removing most if not all instances of floating cameras.

This also fixes any possibility of double-apc rendered issues in Atmos and Southeast Lambda.

All in all, this patch is a very small fix patch that had to happen at some point.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image

image

image

image

image

</details>

# Changelog

:cl: LambdaSucks
fix: Fixed floating cameras in Solaris Ridge inserts. 
fix: Removed second APC from Atmos on Solaris Ridge. 
fix: Removed second APC from Southeast Lambda on Solaris Ridge. 
fix: Fixed floating cameras on Solaris Ridge.
/:cl:

### CODE REVIEW

File: maps/map_files/BigRed/BigRed.dmm [hunk 1]
```diff
@@ -2004,7 +2002,6 @@
 /obj/structure/pipes/standard/simple/hidden/green,
 /obj/structure/machinery/door/airlock/almayer/security/glass/colony{
 	dir = 1;
-	locked = 0;
```
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:2007]: Any reason as to why these doors are no longer locked or ?

File: maps/map_files/BigRed/BigRed.dmm [hunk 2]
```diff
@@ -1992,7 +1991,6 @@
 "afP" = (
 /obj/structure/machinery/door/airlock/almayer/security/glass/colony{
 	dir = 1;
-	locked = 0;
```
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:1995]: Any reason as to why these doors are no longer locked or ?

File: maps/map_files/BigRed/BigRed.dmm [hunk 3]
```diff
@@ -2820,7 +2817,6 @@
 /obj/structure/pipes/standard/simple/hidden/green,
 /obj/structure/machinery/door/airlock/almayer/security/glass/colony{
 	dir = 1;
-	locked = 0;
```
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:2823]: Any reason as to why these doors are no longer locked or ?

File: maps/map_files/BigRed/BigRed.dmm [hunk 4]
```diff
@@ -3398,7 +3394,6 @@
 	dir = 4
 	},
 /obj/structure/machinery/door/airlock/almayer/security/glass/colony{
-	locked = 0;
```
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:3401]: Any reason as to why these doors are no longer locked or ?

File: maps/map_files/BigRed/BigRed.dmm [hunk 5]
```diff
@@ -4689,7 +4684,6 @@
 "anx" = (
 /obj/structure/machinery/door/airlock/almayer/security/glass/colony{
 	dir = 1;
-	locked = 0;
```
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:4692]: Any reason as to why these doors are no longer locked or ?

File: maps/map_files/BigRed/BigRed.dmm [hunk 6]
```diff
@@ -6924,7 +6918,6 @@
 	dir = 1;
 	icon_state = "door_locked";
 	id = "safe_room";
-	locked = 0;
```
Comment (Nanu308) [inline maps/map_files/BigRed/BigRed.dmm:6927]: Any reason as to why these doors are no longer locked or ?

**Review by Nanu308 — CHANGES_REQUESTED** [review]
See comments

File: maps/map_files/BigRed/BigRed.dmm (hunk 1 above)
Comment (LambdaSucks) [reply maps/map_files/BigRed/BigRed.dmm:2007]: Unsure why these were changed. I'll go through and fix them. Should only be a moment.

**Comment by LambdaSucks** [comment]
> See comments

These unlocked doors, and a few others from the Prison Breakout insert have been fixed. Not really sure why they were changed in the first place, but it's been all sorted.

**Review by Nanu308 — APPROVED** [review]
Lgtm

**Pull request merged**